src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
+    private static final int numberOfBalls = 3;
```

Comment:
Please, follow the rules of constants naming:
```suggestion
    private static final int NUMBER_OF_BALLS = 3;
```